Edits to pages in stream ingestion section as part of site review (#194)
pinot-contrib/pinot-docs: basics/data-import/pinot-stream-ingestion/amazon-kinesis.md

* Credentials delivered through the Amazon EC2 container service if `AWS_CONTAINER_CREDENTIALS_RELATIVE_URI` environment variable is set and security manager has permission to access the variable,
* Instance profile credentials delivered through the Amazon EC2 metadata service

> You can also specify the `accessKey` and `secretKey` using the properties. However, this method is not secure and should be used only for POC setups. You can also specify other AWS fields such as AWS\_SESSION\_TOKEN as environment variables and config and it will work.

Reviewer: ```suggestion
Although you can specify the `accessKey` and `secretKey` in properties, we don't recommend this unsecure method. We recommend using only for POC setups. You can also specify other AWS fields such as AWS\_SESSION\_TOKEN as environment variables.
```

Reviewer: where is "properties"?--can we include a link to? 

what are POC setups (non-production/proof of concept)?

#### Limitations

> 1. `ShardID` is of the format "**shardId-000000000001**". We use the numeric part as `partitionId`. Our `partitionId` variable is integer. If `shardIds` grow beyond Integer.MAX\_VALUE, we will overflow

Reviewer: ```suggestion
1. `ShardID` is of the format "**shardId-000000000001**". We use the numeric part as `partitionId`. Our `partitionId` variable is integer. If `shardIds` grow beyond `Integer.MAX\_VALUE`, we will overflow.
```

Reviewer: what does "we will overflow" mean?--can we clarify?

where can this setting be updated?
